README.md

```diff
@@ -1,17 +1,41 @@
 CZDS API Client in Python
 ===========
 
-This repository provides a Python example of how to download zone files via CZDS (Centralized Zone Data Service) REST API. 
+This repository provides a Python example of how to download zone files via CZDS (Centralized Zone Data Service) REST API.
 A detail API Specs can be found [here.](https://github.com/icann/czds-api-client-java/tree/master/docs)
 
-There is also an example provided in Java. It can be found in [this repo.](https://github.com/icann/czds-api-client-java)
+It has been modified to post CloudWatch metrics for each zone file successfully downloaded.
 
 Installation
 ------------
 
-This script requires Python 3. It has been tested with Python 3.7.1. 
+According to ICANN this script requires Python 3, but it works fine with Python 2.7. It has been tested with Python 3.7.1 and 2.7.12.
 
-It requires the `requests` extension library. Please checkout here to see how to install it - https://github.com/requests/requests
+It requires:
+* The `requests` extension library. It can be installed using `pip install requests`.
+* The `boto3` library. This can be installed using `pip install boto3`.
+
+AWS authentication
+------------------
+
+This utility supports authenticating with AWS in the following ways:
+* AWS instance profiles
+* AWS profiles defined in ~/.aws/credentials
+* AWS IAM keys
+
+AWS authentication parameters are configured in `config.yml`. In order to use an instance profile ensure that `aws.iam.profile`, `aws.iam.access_key_id`, and `aws.iam.secret_access_key` are set to `null`. If you want to specify an instance profile just set `aws.iam.profile` to the name of the desired AWS profile you want to use. `aws.iam.access_key_id` and `aws.iam.secret_access_key` should be set to `null` and will be ignored in favor of the AWS authentication profile if specified. In order to use AWS access keys `aws.iam.profile` should be set to `null`.
```

Comment:
It looks like we are mixing references to `config.yml` and `config.json`. There is a `config.sample.json` checked in, so it looks like we just need to consolidate the language.

To this end, I would recommend that we adopt the pattern used [here](https://github.com/legitscript/products-api#running-the-project) for distributing our `config.json` file:

This would be renaming `config.sample.json` to `config.json.dist`.
Then we would unpack the sample `config.json.dist` file using a bash expansion:
`cp config.json{.dist,}`, which removes the `.dist` suffix and results in a file named `config.json`